src/internal/account.ts

```diff
@@ -155,20 +155,20 @@ export async function getResources(args: {
   });
 }
 
-export async function getResource(args: {
+export async function getResource<T extends {}>(args: {
```

Comment:
wonder if we can `T = {}` to default it to `{}` and then dont have to provide any type

Reply:
I went with `T extends {} = any`.  This allows typing to provided by the user, but still allows blind usage via any.